docker-compose.yaml

```diff
@@ -1,32 +1,32 @@
 version: '3'
 services:
-  mine-support:
+  ffc-demo-web:
     build: .
-    image: mine-support
-    container_name: mine-support
+    image: ffc-demo-web
+    container_name: ffc-demo-web
     command: npm run start:watch
     depends_on:
-      - mine-support-redis
+      - ffc-demo-web-redis
     environment:
       COOKIE_PASSWORD: cookiecookiecookiecookiecookiecookie
       MINE_SUPPORT_API_GATEWAY: http://mine-support-api-gateway:3001

```

Comment:
Missed renaming this `MINE_SUPPORT_API_GATEWAY` env var

Reply:
Thanks.  Fixed!